api_migration.md

```diff
@@ -0,0 +1,36 @@
+# Changes to the api
```

Comment:
Thank you for adding this.

There is also a code -> code_id renaming from another PR I believe

Reply:
It was `code_id` in the json before. Only the Go code was inconsistent. There are no changes for the clients